dev_tools/requirements/envs/dev.env.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
please use the docker image to compile the requirements

Reply:
done